app/scenes/Document/components/SharePopover.tsx

```diff
@@ -47,22 +49,23 @@ function SharePopover({
   const canPublish =
     can.update &&
     !document.isTemplate &&
-    auth.team?.sharing &&
+    team.sharing &&
     documentAbilities.share;
   const isPubliclyShared =
-    (share && share.published) ||
-    (sharedParent && sharedParent.published && !document.isDraft);
+    team.sharing &&
```

Comment:
I considered adding this to the presenter, but this would mean reloading all of the share records cached in the client when the team setting changes